subprojects/core/src/main/java/org/gradle/api/internal/tasks/properties/OutputFilePropertyType.java

```diff
@@ -32,6 +34,7 @@
         this.validationAction = validationAction;
     }
 
+    @UsedByScanPlugin("test-distribution")
```

Comment:
This is not good. We shouldn't have these dependencies. /cc @britter @jonnybbb 

Not for this PR to fix.